lib/not-found.js

```diff
@@ -31,7 +31,8 @@ module.exports = options => {
 
     res.status(404).render('404', {
       title: translations.title,
-      description: translations.description
+      description: translations.description,
+      startLink: req.path.replace(/^\/([^\/]*).*$/, '$1')
```

Comment:
It looks like it's stripping the leading slash off the path, which possibly leads to my next question... why?